interop-testing/src/main/java/io/grpc/testing/integration/GrpclbFallbackTestClient.java

```diff
@@ -32,35 +32,32 @@
 import java.util.logging.Logger;
 
 /**
- * Test client that verifies that grpclb failover into fallback mode works under
- * different failure modes.
- * This client is suitable for testing fallback with any "grpclb" load-balanced
- * service, but is particularly meant to implement a set of test cases described
- * in an internal doc titled "DirectPath Cloud-to-Prod End-to-End Test Cases",
- * section "gRPC DirectPath-to-CFE fallback".
+ * Test client that verifies that grpclb failover into fallback mode works under different failure
+ * modes. This client is suitable for testing fallback with any "grpclb" load-balanced service, but
+ * is particularly meant to implement a set of test cases described in an internal doc titled
+ * "DirectPath Cloud-to-Prod End-to-End Test Cases", section "gRPC DirectPath-to-CFE fallback".
  */
 public final class GrpclbFallbackTestClient {
-  private static final Logger logger =
-      Logger.getLogger(GrpclbFallbackTestClient.class.getName());
+  private static final Logger logger = Logger.getLogger(GrpclbFallbackTestClient.class.getName());
 
-  /**
-   * Entry point.
-   */
+  /** Entry point. */
```

Comment:
nit: Can you please revert unrelated changes to make it easier to review?

Reply:
The style change is because I used `google-java-format-1.9-all-deps.jar` to format the code. Maybe let's keep the change as it obeys the style guidance?